diff --git a/node_modules/@docusaurus/theme-search-algolia/lib/theme/SearchBar/index.js b/node_modules/@docusaurus/theme-search-algolia/lib/theme/SearchBar/index.js
index 4e5b33d..6c2894b 100644
--- a/node_modules/@docusaurus/theme-search-algolia/lib/theme/SearchBar/index.js
+++ b/node_modules/@docusaurus/theme-search-algolia/lib/theme/SearchBar/index.js
@@ -86,6 +86,7 @@ function DocSearch({contextualSearch, externalUrlRegex, ...props}) {
   const onClose = useCallback(() => {
     setIsOpen(false);
     searchContainer.current?.remove();
+    searchButtonRef.current?.focus();
   }, [setIsOpen]);
   const onInput = useCallback(
     (event) => {
